docs/get-started/sveltekit.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Storybook for SvelteKit
+---
+
+export const SUPPORTED_RENDERER = 'svelte';
+
+Storybook for SvelteKit is a [framework](../contribute/framework.md) that makes it easy to develop and test UI components in isolation for [SvelteKit](https://kit.svelte.dev/) applications. It includes:
+
+- 🪄 Zero config
+- 🧩 Easily mock many Kit modules
+- 🔗 Automatic link handling
+- 💫 and more!
+
+<If notRenderer={SUPPORTED_RENDERER}>
+
+<Callout variant="info">
+
+Storybook for SvelteKit is only supported in [Svelte](?renderer=svelte) projects.
+
+</Callout>
+
+<!-- End non-supported renderers -->
+
+</If>
+
+<If renderer={SUPPORTED_RENDERER}>
+
+## Requirements
+
+- SvelteKit ≥ 1.0 (not including beta versions)
```

Comment:
I don't think we need to state the part about beta versions anymore, they are long gone.

```suggestion
- SvelteKit ≥ 1.0
```